### DISCUSSION TITLE
Laravel, Swoole 6.0 and Multithreading!  It seems now we have this option!

### DISCUSSION THREAD

**Discussion by proilyxa** [swoole/swoole-src#5650] [Ideas]
Guys, I spent the evening with swoole 6.0 and Laravel, and even wrote a small library for working with multithreading. Share your opinion, do you think Laravel implements something like this natively?

https://github.com/proilyxa/lara-thread

**Comment by nguyenhothanhtam0709** [comment]
@proilyxa, can you run Laravel Octane with Swoole 6.0?

**Reply by proilyxa** [reply]
yes

**Reply by nguyenhothanhtam0709** [reply]
Due to issue [5651](https://github.com/swoole/swoole-src/issues/5651), currently Octane will raise error if Swoole is running with SWOOLE_THREAD. Also, I just discovered that Octane using on task callback that isn't compatible with Swoole `task_enable_coroutine`
